pkgs/by-name/ed/edmarketconnector/package.nix

```diff
@@ -0,0 +1,81 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  python3Full,
+  makeWrapper,
+  makeDesktopItem,
+}:
+let
+  pkgVersion = "5.12.2";
+  pythonEnv = python3Full.buildEnv.override {
+    extraLibs = with python3Full.pkgs.pythonPackages; [
+      requests
+      pillow
+      (watchdog.overrideAttrs {
+        disabledTests = [
+          "test_select_fd"
+        ];
+      })
+      semantic-version
+      psutil
+    ];
+    ignoreCollisions = true;
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "edmarketconnector";
+  version = "${pkgVersion}";
+
+  src = fetchFromGitHub {
+    owner = "EDCD";
+    repo = "EDMarketConnector";
+    tag = "Release/${pkgVersion}";
+    hash = "sha256-3ywu/EJdIsKqTN3uaA5F0tZK6tybl483Yiwqh7W4yCc=";
+  };
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "EDMarketConnector";
```

Comment:
I think its more "correct" to name it `E:D Market Connector` since this is the official name from the top of the application itself and the official desktop file.
https://github.com/EDCD/EDMarketConnector/blob/2f3f7262da81f4892972560822662e9205c7997c/io.edcd.EDMarketConnector.desktop#L6
![image](https://github.com/user-attachments/assets/28aa1789-42d8-496e-9e29-7e0b56b709ee)

For that matter, why even make a desktop item up here, when the original from the `src` is getting modified and also used in `installPhase`?